qosf.org/scripts/github_scraping.py

```diff
@@ -18,7 +18,8 @@
 import yaml
 import requests
 
-readme = requests.get('https://raw.githubusercontent.com/qosf/os_quantum_software/master/README.md')
+readme = requests.get('https://raw.githubusercontent.com/CalMacCQ/awesome-quantum-software/master/README.md', timeout=5)
```

Comment:
This is a request to my branch on the awesome-quantum-software repo. Will replace with the proper https one the fix is merged.

Reply:
Updated to `qosf/awesome-quantum-software` rather than my fork.